roles/aap_setup_prepare/templates/inventory.j2

```diff
@@ -0,0 +1,19 @@
+{# output all the hosts and their groups #}
+
+{% for group_key in aap_setup_prep_inv_nodes %}
+[{{ group_key }}]
+{% for node_key in aap_setup_prep_inv_nodes[group_key] %}
+{{ node_key }} {{ aap_setup_prep_inv_nodes[group_key][node_key] }}
```

Comment:
This line needs tweaking to
```
{{ node_key }}{% if aap_setup_prep_inv_nodes[group_key][node_key] is defined %} {{ aap_setup_prep_inv_nodes[group_key][node_key] }}{% endif %}

```
So that if a user defines it as such:
```
aap_setup_prep_inv_nodes:
  automationcontroller:
    controller.node: "servicescatalog_controller_name=something"
  database:
    database
  automationhub:
    ah.node
```
It doesn't complain about it not being there.

Reply:
Done!